config/routes.rb

```diff
@@ -36,6 +36,8 @@
   get '/our_services', controller: 'home', action: 'services'
   get '/resources', controller: 'home', action: 'resources'
   get '/training', controller: 'home', action: 'training'
+  get '/project_requirements', controller: 'home', 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.